components/single_image_view/single_image_view.test.tsx

```diff
@@ -106,4 +106,27 @@ describe('components/SingleImageView', () => {
         expect(wrapper.find(SizeAwareImage).prop('handleSmallImageContainer')).
             toEqual(true);
     });
+
+    test('should not show filename when image is displayed', () => {
+        const wrapper = shallow(
+            <SingleImageView
+                {...baseProps}
+                isEmbedVisible={true}
+            />,
+        );
+
+        expect(wrapper.find('[data-testid="image-name"]')).toHaveLength(0);
```

Comment:
nit: only checking against an element not being there is sometimes a bit misleading, since not rendering it due to some kind of error would pass this test as well. It would be best to pair this with a test that renders the container or something that should be there regardless of the filename. Thoughts?

Reply:
Sure, totally makes sense! Updated the test to check text within the parent container, which should always exist (and the test will fail if it doesn't).